auto-update-repository.py

```diff
@@ -25,8 +104,26 @@ def create_package(repository: Dict[str, Any], payload: Dict[str, str]) -> None:
     update_package(new_package, payload)
 
 
+def set_workflow_output(**kwargs: str) -> None:
+    for key, value in kwargs.items():
+        print("::set-output name={}::'{}'".format(key, value))
```

Comment:
Untested :) I hope this works to set outputs to be used in a next workflow step.